app/lib/katello/agent/client_message_handler.rb

```diff
@@ -0,0 +1,63 @@
+module Katello
+  module Agent
+    class ClientMessageHandler
+      def self.logger
+        ::Foreman::Logging.logger('katello/agent')
+      end
+
+      # rubocop:disable Metrics/MethodLength
+      def self.handle(message)
```

Comment:
can you break this up to smaller methods.

Reply:
Let me think about that. I'm all for small methods but I'm not sure those two suggested methods serve their own individual purpose